### ISSUE TITLE
Stop using babel-node in production

### ISSUE THREAD

**Issue by ch2ch3** [ustwo/ustwo.com-frontend#258]
- [ ] Update compilation scripts to compile the server code
- [ ] Make sure Docker runs the compiled scripts

**Comment by daaain** [comment]
Blocked by https://github.com/ustwo/ustwo.com-frontend/issues/222 and https://github.com/ustwo/ustwo.com-frontend/pull/235

**Comment by kater1** [comment]
No longer blocked by #235

**Comment by daaain** [comment]
Since everything from our application server is behind CDN cache now (meaning it'll be hit on average once an hour for the CDN to source new content, plus manual purges) and the biggest penalty is only on startup time, we decided in backlog grooming to close this due to low priority unless we see some issues. Hopefully Node will catch up with ES6 features and we can just switch over without having to put in this transpiling step in place.
